src/main/java/com/adyen/model/recurring/ScheduleAccountUpdaterRequest.java

```diff
@@ -0,0 +1,177 @@
+        this.merchantAccount = merchantAccount;
+        return this;
+    }
+
+    public String getMerchantAccount() {
+        return merchantAccount;
+    }
+
+    public void setMerchantAccount(String merchantAccount) {
+        this.merchantAccount = merchantAccount;
+    }
+
+    public ScheduleAccountUpdaterRequest reference(String reference) {
+        this.reference = reference;
+        return this;
+    }
+
+    public String getReference() {
+        return reference;
+    }
+
+    public void setReference(String reference) {
+        this.reference = reference;
+    }
+
+    public ScheduleAccountUpdaterRequest selectedRecurringDetailReference(String selectedRecurringDetailReference) {
+        this.selectedRecurringDetailReference = selectedRecurringDetailReference;
+        return this;
+    }
+
+    public String getSelectedRecurringDetailReference() {
+        return selectedRecurringDetailReference;
+    }
+
+    public void setSelectedRecurringDetailReference(String selectedRecurringDetailReference) {
+        this.selectedRecurringDetailReference = selectedRecurringDetailReference;
+    }
+
+    public ScheduleAccountUpdaterRequest shopperReference(String shopperReference) {
+        this.shopperReference = shopperReference;
+        return this;
+    }
+
+    public String getShopperReference() {
+        return shopperReference;
+    }
+
+    public void setShopperReference(String shopperReference) {
+        this.shopperReference = shopperReference;
+    }
+
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
+        ScheduleAccountUpdaterRequest scheduleAccountUpdaterRequest = (ScheduleAccountUpdaterRequest) o;
+        return Objects.equals(this.additionalData, scheduleAccountUpdaterRequest.additionalData) &&
+                Objects.equals(this.card, scheduleAccountUpdaterRequest.card) &&
+                Objects.equals(this.merchantAccount, scheduleAccountUpdaterRequest.merchantAccount) &&
+                Objects.equals(this.reference, scheduleAccountUpdaterRequest.reference) &&
+                Objects.equals(this.selectedRecurringDetailReference, scheduleAccountUpdaterRequest.selectedRecurringDetailReference) &&
+                Objects.equals(this.shopperReference, scheduleAccountUpdaterRequest.shopperReference);
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(additionalData, card, merchantAccount, reference, selectedRecurringDetailReference, shopperReference);
+    }
+
+
+    @Override
+    public String toString() {
+        StringBuilder sb = new StringBuilder();
+        sb.append("class ScheduleAccountUpdaterRequest {\n");
+
+        sb.append("    additionalData: ").append(toIndentedString(additionalData)).append("\n");
+        sb.append("    card: ").append(toIndentedString(card)).append("\n");
```

Comment:
sensitive information- card?

Reply:
In this case the `card` is the card object itself, so there should be no need to mask that (since card object itself is masked with https://github.com/Adyen/adyen-java-api-library/pull/421)